src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/ContainerNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+public class ContainerNotFoundException extends RuntimeException {
+
+    private static final long serialVersionUID = -3411204344653345638L;
+
+    public ContainerNotFoundException() {
```

Comment:
Perhaps would be handy for us to know which container is it and then we can stick in instead of `BLOB`?

Reply:
> Perhaps would be handy for us to know which container is it and then we can stick in instead of `BLOB`?

delete the exception